src/Calendar.js

```diff
@@ -42,11 +42,19 @@ const Calendar = ({
       /* istanbul ignore else */
       if (key === 'Tab') calendarElement.current.classList.remove('-noFocusOutline');
     };
-    calendarElement.current.addEventListener('keyup', handleKeyUp, false);
+
+    /* istanbul ignore else */
+    if (calendarElement.current !== null) {
+      calendarElement.current.addEventListener('keyup', handleKeyUp, false);
+    }
+
     return () => {
-      calendarElement.current.removeEventListener('keyup', handleKeyUp, false);
+      /* istanbul ignore else */
+      if (calendarElement.current !== null) {
+        calendarElement.current.removeEventListener('keyup', handleKeyUp, false);
+      }
     };
-  });
+  }, [calendarElement]);
```

Comment:
this will not take any effect, because useRef (line 32) just return the same object instance on every render. I agree that this fix (i mean check for null) resolve issue with throwing an exception, but I disagree that it resolves the root of the problem.
I suggest to save reference value inside the useEffect hook and operate on it, instead of accessing reference current value.
```
useEffect(() => {
  const element = calendarElement.current;
  const handleKeyUp = ({ key }) => {
    /* istanbul ignore else */
    if (key === 'Tab') calendarElement.current.classList.remove('-noFocusOutline');
  };
  element.addEventListener('keyup', handleKeyUp, false);
  return () => {
    element.removeEventListener('keyup', handleKeyUp, false);
  };
});
```